src/class/hid/hid_device.c

```diff
@@ -67,20 +67,28 @@ static inline hidd_interface_t* get_interface_by_itfnum(uint8_t itf_num)
   return NULL;
 }
 
+static inline uint8_t get_hid_index_by_itfnum(uint8_t itf_num)
+{
+	for (uint8_t i=0; i < CFG_TUD_HID; i++ )
+	{
+		if ( itf_num == _hidd_itf[i].itf_num ) return i;
+	}
+
+	return 0;
```

Comment:
0 is valid return, you should return either -1 or 0xff and check the return in the caller. But it is ok to leave it as it is, I will merge the usage of `get_hid_index_by_itfnum & get_interface_by_itfnum` as follow up PR as well.